src/components/account.tsx

```diff
@@ -0,0 +1,196 @@
+        const { balances, selectedAccount } = this.state;
+        const accountBalances = balances[selectedAccount];
+        const fetchBalancesButton = (
+            <Button isSize="small" isColor="info" id="fetchAccountBalances" onClick={this.fetchAccountDetailsAsync}>
+                Fetch Balances
+            </Button>
+        );
+        let contentRender = (
+            <div>
+                <strong>Fetching Balances...</strong>
+            </div>
+        );
+
+        if (accountBalances) {
+            const balancesString = _.map(accountBalances, (tokenBalance: TokenBalanceAllowance) => {
+                const { name, symbol } = tokenBalance.token;
+                // Convert to the human readable amount based off the token decimals
+                const balance = Web3Wrapper.toUnitAmount(tokenBalance.balance, tokenBalance.token.decimals);
+                const balanceRender = balance.toFixed(4);
+                const allowanceRender = this.renderAllowanceForTokenBalance(tokenBalance);
+                return (
+                    <tr key={name}>
+                        <td>{symbol}</td>
+                        <td>{balanceRender}</td>
+                        <td>{allowanceRender}</td>
+                    </tr>
+                );
+            });
+            contentRender = (
+                <div>
+                    <Table>
+                        <thead>
+                            <tr>
+                                <th>Token</th>
+                                <th>Balance</th>
+                                <th>Allowance</th>
+                            </tr>
+                        </thead>
+                        <tbody>{balancesString}</tbody>
+                    </Table>
+                </div>
+            );
+        }
+
+        return (
+            <Content style={{ marginTop: '15px' }}>
+                Below you will find the Account and token balances. The lock icon ({' '}
+                <Icon isSize="small" className="fa fa-lock" />) indicates that this token is not tradeable on 0x, to
+                unlock the token click the lock icon. The tick icon ({' '}
+                <Icon isSize="small" className="fa fa-check-circle" style={{ color: 'rgb(77, 197, 92)' }} /> ) indicates
+                that this token is tradeable on 0x.
+                <Subtitle isSize={6}>Account: {selectedAccount}</Subtitle>
+                <Columns>
+                    <Column isSize={3}>{contentRender}</Column>
+                </Columns>
+                {fetchBalancesButton}
+            </Content>
+        );
+    }
+
+    public renderAllowanceForTokenBalance(tokenBalance: TokenBalanceAllowance): React.ReactNode {
+        let allowanceRender;
+        if (tokenBalance.tradeable) {
+            allowanceRender = tokenBalance.allowance.greaterThan(0) ? (
+                <Icon isSize="small" className="fa fa-check-circle" style={{ color: 'rgb(77, 197, 92)' }} />
```

Comment:
Let's set `rgb(77, 197, 92)` and other color RGB/Hex strings with names that elucidate the color it encodes. e.g `const LIGHT_GRAY = 'rgb(77, 197, 92)';`